ci/dhis2-verify-app.yml

```diff
@@ -83,31 +83,22 @@ jobs:
     #          with:
     #              node-version: 12.x
     #
-    #        - uses: c-hive/gha-yarn-cache@v1
-    #        - run: yarn install --frozen-lockfile
-    #
-    #        - name: Install Cypress binary
-    #          run: yarn cypress install
-    #
     #        - name: End-to-End tests
     #          uses: cypress-io/github-action@v2
     #          with:
-    #              install: false
+    #              # This should be a command that serves the app without opening a browser.
+    #              # For example: `BROWSER=none d2-app-scripts start`
+    #              start: yarn start:nobrowser
+    #              wait-on: 'http://localhost:3000'
+    #              wait-on-timeout: 300
     #              record: true
     #              parallel: true
-    #              start: ${{ env.SERVER_START_CMD }}
-    #              wait-on: ${{ env.SERVER_URL }}
-    #              wait-on-timeout: 300
-    #              cache-key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-    #              group: 'e2e'
-    #              tag: ${{ github.event_name }}
     #          env:
-    #              CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
     #              GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    #              COMMIT_INFO_MESSAGE: ${{ github.event.pull_request.title }}
```

Comment:
Very naive question: why don't we need `COMMIT_INFO_MESSAGE` anymore?

Reply:
I've stuck to the bare minimum that was needed for the cypress action. It (https://github.com/cypress-io/github-action) doesn't require this, so I've just removed it.